src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1308,4 +1308,7 @@ Please specify one of them using the `--source` option to proceed.</value>
   <data name="NoPackageSelectionArgumentProvided" xml:space="preserve">
     <value>No package selection argument was provided; see the help for details about finding a package.</value>
   </data>
+  <data name="ListAllArgumentDescription" xml:space="preserve">
+    <value>List all software installed by a package</value>

```

Comment:
nit: we try to make "package" the terminology for software across winget but I can see it's weird to say "all packages installed by a package"

Maybe "List all package components installed by a package" or "List all sub-packages installed by a package"

Open to any suggestions.

Reply:
@denelon, thoughts? I think "List all components installed by a package makes the most sense".

Reply:
I refer to these as "system artifacts" internally, but that probably isn't a great name for them publicly.  I do like "components" or "items".